CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 2022-06-a
+- consume XML files with queries captured by SQL Server Profiler.
```

Comment:
nit: A styling comment. The rest of the changelog starts each sentence capitalized. Suggest we follow the same format.

Reply:
done